core/templates/dev/head/pages/exploration_editor/simple_editor_tab/sidebarHighlightingService.js

```diff
@@ -0,0 +1,46 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Service that highlights the color of the question
+ * depending on wheather it is finished or unfinished .
```

Comment:
This is somehow misleading since it only provides functionality to do the highlighting.